.github/workflows/lkg-on-release-push.yml

```diff
@@ -0,0 +1,52 @@
+name: Update LKG on release branch push
+
+on:
+  push:
+    branches:
+      - 'release-*'
+    branches-ignore:
+      - 'release-[0-4].*' # Predates hereby
```

Comment:
Apparently you can't have both of these 
![image](https://github.com/microsoft/TypeScript/assets/5341706/549550dd-c96c-4027-b147-d9bf1bbf47ce)